driver-mapping/src/main/java/com/datastax/driver/mapping/configuration/naming/NamingStrategy.java

```diff
@@ -0,0 +1,130 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.mapping.configuration.naming;
+
+public class NamingStrategy {
+
+    private boolean enabled;
+
+    private NamingConvention javaConvention;
+
+    private NamingConvention cassandraConvention;
```

Comment:
> disable naming strategy all together

I don't think users would actually want that, there should always be a naming strategy in place.

The question is: what should be the _default_ strategy? I think we need to prioritize continuity and favor a strategy that actually yields the same results as currently, _without any need to customize the default settings_. In other words: favor convention over configuration. So I think indeed we need a "passthrough" strategy as our default.